Source/Scene/Model.js

```diff
@@ -772,6 +781,10 @@ define([
         return JSON.parse(json);
     }
 
+    Model.getDefaultCacheKey = function(url) {
```

Comment:
At the very least, it should be `Model._getDefaultCacheKey` and be marked `@private`.  But why not just have `ModelInstanceCollection` access the `_url` property directly?  Can't `_url` be `undefined` if the model was loaded from a JSON object?